team.html

```diff
@@ -64,6 +64,7 @@ <h2 id="overview"><a href="#overview"> Composition </a></h2>
             <li><a href="#project-manager">Project managers</a></li>
             <li><a href="#lead-developer">Lead developers</a></li>
             <li><a href="#module-maintainer">Sub-package maintainers</a></li>
```

Comment:
Do we still retain sub-package maintainers?

Reply:
Good question... I can comment out this section, indeed.

However, searching and having new sub-packages maintainers can be good for us. If you have any suggestion, one can discuss about that on the lead-dev slack channel